SYCL/ESIMD/api/functional/ctors/ctor_fill_fp_extra.cpp

```diff
@@ -0,0 +1,50 @@
+//==------- ctor_fill_fp_extra.cpp  - DPC++ ESIMD on-device test -----------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu, level_zero
+// XREQUIRES: gpu
+// TODO gpu and level_zero in REQUIRES due to only this platforms supported yet.
+// The current "REQUIRES" should be replaced with "gpu" only as mentioned in
+// "XREQUIRES".
+// UNSUPPORTED: cuda, hip
+// XRUN: %clangxx -fsycl %s -fsycl-device-code-split=per_kernel -o %t.out
+// XRUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: false
+// XFAIL: *
+// TODO The simd can't be constructed with sycl::half data type. The issue was
+// created (https://github.com/intel/llvm/issues/5077) and the this test must be
```

Comment:
```suggestion
// created (https://github.com/intel/llvm/issues/5077) and this test must be
```

Reply:
may be this statement will be better `and the test must be`?
I've use it [here](https://github.com/intel/llvm-test-suite/pull/598/commits/1c807fc362443314f1e18c19bc469cffaef58e2b#diff-363883cae3ba91f2ba9d07628c42317d4da5a038cb6bef71d8f3023281b6cc79L19-L20)

Reply:
Good for me either way, thanks)